src/pages/Profile/Profile.tsx

```diff
@@ -4,9 +4,36 @@ import { Link } from "react-router-dom";
 import { UserProfile } from "../../mockData/mockTribe";
 import Header from "../../components/Header/Header";
 import Navigation from "../../components/Navigation/Navigation";
+import { signOut } from "firebase/auth";
+import { auth } from "../../firebase";
+import { useNavigate } from "react-router-dom";
 
-const Profile = ({ user }: { user: UserProfile }) => {
+type profileProps = {
+  setUserUID: (userUID: string | null) => void;
+  user: UserProfile;
+};
+
+const Profile = ({ setUserUID, user }: profileProps) => {
   const { totalScore, img, name, bio, email } = user;
+  const navigate = useNavigate();
+
+  const handleSignOut = async () => {
+  
+    try {signOut(auth);
+        navigate("/auth");
+        localStorage.removeItem("userData");
```

Comment:
The value that gets added to local storage is "accessToken" so you will want to remove that.

localStorage.setItem("accessToken", JSON.stringify(accessToken))